pkg/controller/operator.go

```diff
@@ -72,7 +75,12 @@ func (s *Operator) Run(ctx context.Context, controller *controllercmd.Controller
 	}
 	configInformers := configv1informers.NewSharedInformerFactory(configClient, 10*time.Minute)
 
-	operatorClient, err := operatorv1client.NewForConfig(controller.KubeConfig)
+	operatorClientV1, err := operatorv1client.NewForConfig(controller.KubeConfig)
+	if err != nil {
+		return err
+	}
```

Comment:
I think we don't necessarily need this `operatorClientV1`. You can use the below created `operatorClient` instead.

Reply:
thanks for the review, done!